### PR TITLE
Fix admin link

### PR DESCRIPTION
**Pull request by rarmitag**
*Issue #:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: components/app_header/Container.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
       </div>
 
       <div class="ml-3 flex gap-10 text-bcgov-blue5">
-        <AppHeaderNavLink text="Admin" :route="Route.Admin" />
+        <AppHeaderNavLink text="Admin" :route=adminURL />
```
Comment (semmatti) [inline components/app_header/Container.vue:18]: Make the link open in a new tab
```suggestion
        <AppHeaderNavLink text="Admin" :route=adminURL target="_blank"/>
```

**Review by semmatti — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namex-dev--pr-1431-u0cytkyc.web.app

**Pull request merged**